gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/ApiExportServiceImpl.java

```diff
@@ -139,6 +143,10 @@ public String exportAsCustomResourceDefinition(ExecutionContext executionContext
                 apiDefinitionResource.setVersion(exportQuery.getVersion());
             }
 
+            ((ObjectNode) jsonNode).set("notifyMembers", BooleanNode.valueOf(!apiEntity.isDisableMembershipNotifications()));
```

Comment:
Why not just reading the value from the json node that you remove on next line ? It could allow you to avoid calling the findById.

Reply:
For some reason the value does not get serialised but you are right, I'll give it another go

Reply:
It's done and fixes this in JSON export as well